### PR TITLE
Adding startup logic where -paytxfee if lower than default, mintxfee is overridden

### PR DESCRIPTION
**Pull request by michilumin**

### CODE REVIEW

**Comment by polska333** [comment]
What's new with doge any updates coming will they be burning tokens
something has to be done about the supply it's too big and people will stop
Ferguson if the price does not go up that's not me to be honest I can't
even afford to buy any I'm just a fan I'm disabled and I eat through a
feeding tube a year ago I was even dead before I died I was telling people
to invest in dogecoin I made a lot of people a lot of money I got sick and
everything and they also got about me. That's how greedy people are I love
those coin I didn't get a chance to invest because I literally died and now
I'm in an assisted living facility eating through a feeding tube

**Comment by KapitalEarth** [comment]
The fixed # of coins per unit of time will remain and no Dogecoin will be burnt. An update is imminent (~days), thanks to the awesome work of the dev team. 

Sorry to hear of your health complications. Wishing you a successful recovery.

**Comment by polska333** [comment]
I kind of figured that the coins are going to remain the same you hit
different stuff everywhere from people anyways I'm a huge fan and keep up
the good work

File: src/wallet/wallet.cpp [hunk 1]
```diff
@@ -3894,6 +3894,13 @@ bool CWallet::ParameterInteraction()
             return InitError(strprintf(_("Invalid amount for -paytxfee=<amount>: '%s' (must be at least %s)"),
                                        GetArg("-paytxfee", ""), ::minRelayTxFeeRate.ToString()));
         }
+	// if -mintxfee is not set, then a lower payTxFee overrides minTxFee
+	if (!IsArgSet("-mintxfee") && payTxFee < CWallet::minTxFee)
+        {
```
Comment (patricklodder) [inline src/wallet/wallet.cpp:3899]: nit: could use an indent fix

File: src/wallet/wallet.cpp [hunk 2]
```diff
@@ -3894,6 +3894,13 @@ bool CWallet::ParameterInteraction()
...
+            InitWarning(strprintf(_("Automatically lowering -mintxfee to '%s'"), GetArg("-paytxfee","")));
```
Comment (patricklodder) [inline src/wallet/wallet.cpp:3900]: the `InitWarning()` needs to be removed for this to be testable with rpc-tests

**Review by langerhans — APPROVED** [review]

**Comment by polska333** [comment]
What's the newest updates to dogecoin right now any big ones

**Pull request merged**